sentry-kotlin-multiplatform/src/commonAppleMain/kotlin/io/sentry/kotlin/multiplatform/nsexception/SentryUnhandledExceptions.kt

```diff
@@ -66,19 +64,19 @@ internal fun setSentryUnhandledExceptionHook(): Unit = wrapUnhandledExceptionHoo
 /**
  * Tag used to mark the Kotlin termination crash.
  */
-private const val kotlinCrashedTag = "nsexceptionkt.kotlin_crashed"
+internal const val kotlinCrashedTag = "nsexceptionkt.kotlin_crashed"
 
 /**
  * Converts `this` [Throwable] to a [SentryEnvelope].
  */
-private fun Throwable.asSentryEnvelope(): SentryEnvelope {
+internal fun Throwable.asSentryEnvelope(): SentryEnvelope {
     val event = asSentryEvent()
     val preparedEvent = SentrySDK.currentHub().let { hub ->
         hub.getClient()?.prepareEvent(event, hub.scope, alwaysAttachStacktrace = false, isCrashEvent = true)
     } ?: event
     val item = SentryEnvelopeItem(preparedEvent)
     // TODO: pass traceState when enabling performance monitoring for KMP SDK
-    val header = SentryEnvelopeHeader(preparedEvent.eventId, SentrySDK.options?.sdkInfo, null)
+    val header = SentryEnvelopeHeader(preparedEvent.eventId, null)
```

Comment:
M: why is the sdkInfo removed?

Reply:
Does that mean that events dont have the SDK info anymore?

Reply:
it's removed because the cocoa sdk deprecated sdkInfo. So `SentryEnvelopeHeader` does not have a param in the constructor with `sdkInfo` anymore.

The sdkInfo is still being set in `beforeSend` with `PrivateSentrySDKOnly.setSdkName(sdkName, sdkVersion)`